examples/profile/profile.html

```diff
@@ -19,47 +20,76 @@ <h1><a href="/">Home</a></h1>
         <td>Selfie from saved Image</td>
         <td><img src="/images/YotiSelfie.jpeg" /></td>
       </tr>
+      {{end}}
+      {{if .profile.GivenNames}}
       <tr>
         <td>First Name:</td>
         <td>{{.profile.GivenNames}}</td>
       </tr>
+      {{end}}
+      {{if .profile.FamilyName}}
       <tr>
         <td>Family Name:</td>
         <td>{{.profile.FamilyName}}</td>
       </tr>
+      {{end}}
+      {{if .profile.FullName}}
       <tr>
         <td>Full Name:</td>
         <td>{{.profile.FullName}}</td>
       </tr>
+      {{end}}
+      {{if .profile.MobileNumber}}
       <tr>
         <td>Mobile Number:</td>
         <td>{{.profile.MobileNumber}}</td>
       </tr>
+      {{end}}
+      {{if .profile.EmailAddress}}
       <tr>
         <td>Email Address:</td>
         <td>{{.profile.EmailAddress}}</td>
       </tr>
+      {{end}}
+      {{if .profile.DateOfBirth}}
       <tr>
         <td>Date of Birth:</td>
         <td>{{.profile.DateOfBirth}}</td>
       </tr>
+      {{end}}
+      {{if .profile.IsAgeVerified}}
       <tr>
         <td>Is Age Verified:</td>
         <td>{{.profile.IsAgeVerified}}</td>
       </tr>
+      {{end}}
+      {{if .profile.Address}}
       <tr>
         <td>Address:</td>
         <td>{{.profile.Address}}</td>
       </tr>
+      {{end}}
+      {{if .profile.Gender}}
       <tr>
         <td>Gender:</td>
         <td>{{.profile.Gender}}</td>
       </tr>
+      {{end}}
+      {{if .profile.Nationality}}
       <tr>
         <td>Nationality:</td>
         <td>{{.profile.Nationality}}</td>
       </tr>
-
+      {{end}}
+      {{if .profile.StructuredPostalAddress}}
```

Comment:
that `<td>` should be inside a `<tr>` otherwise it breaks html semantic - imo.
I would have written it this way:

{{if .profile.StructuredPostalAddress}}
<tr>
	<td>Structured Postal Address:</td>
	{{range $key, $value := .profile.StructuredPostalAddress}} 
         <td>
                <table>      
	               <tr>  
	                      <td>{{$key}}</td>
	                      <td>{{$value}}</td>
	                </tr>  
                  </table>   
            </td>     
	{{end}}
</tr>
{{end}}